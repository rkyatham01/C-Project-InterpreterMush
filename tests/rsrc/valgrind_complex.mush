10 echo "Let's start!"
20 set x = 0
30 sleep 1
40 set x = #x + 1
50 if #x <= 3 goto 30
60 echo #x
run

