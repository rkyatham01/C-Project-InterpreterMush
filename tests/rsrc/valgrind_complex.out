Let's start!
4
